### PR TITLE
:ambulance: correct xpath for static links

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

**Review by Ramil-Mukhametzyanov — APPROVED** [review]
The error of selecting a product at POS is fixed.

**Review by yelizariev — CHANGES_REQUESTED** [review]
No version update

File: pos_order_cancel_restaurant/doc/changelog.rst [hunk 1]
```diff
@@ -1,3 +1,8 @@
+`1.4.2`
+-------
+
+- **FIX:** Multi-Session methods renamings
```
Comment (yelizariev) [inline pos_order_cancel_restaurant/doc/changelog.rst]: WHat is it?

Comment (GabbasovDinar) [reply pos_order_cancel_restaurant/doc/changelog.rst]: @yelizariev DONE

**Review by yelizariev — APPROVED** [review]

**Pull request merged**